gravitee/templates/api-deployment.yaml

```diff
@@ -30,14 +30,12 @@ spec:
           ports:
             - containerPort: {{ .Values.api.service.internalPort }}
           livenessProbe:
-            httpGet:
-              path: /management/apis
+            tcpSocket:
```

Comment:
any specific reason to move to tcpSocket?

Reply:
Since we are using Nginx Ingress Controller and managing routes through the Controller, we have the ability to configure which service will be served in which port. For the livenessProbe, I thought checking just the port should suffice. I had problems when using the previous version as the container was never ready because the livenessProbe was never "live". With the port, it works 100% of the time.

Reply:
still these are http-based services, so the more natural probes, it looks `httpGet` to me :)